terraform/stacks/main/outputs.tf

```diff
@@ -103,30 +103,30 @@ output "services_subnet_az2" {
   value = "${module.cf.services_subnet_az2}"
 }
 output "services_subnet_cidr_az1" {
-  value = "${var.services_cidr_1}"
+  value = "${module.cf.services_cidr_1}"
```

Comment:
Looks like the `cloudfoundry` module doesn't define an output with this name--we'll need to add one.